conntrack_linux.go

```diff
@@ -641,9 +637,10 @@ func parseRawData(data []byte) *ConntrackFlow {
 		if nested, t, l := parseNfAttrTL(reader); nested {
 			switch t {
 			case nl.CTA_TUPLE_ORIG:
-				if nested, t, l = parseNfAttrTL(reader); nested && t == nl.CTA_TUPLE_IP {
+				if nested, t, _ = parseNfAttrTL(reader); nested && t == nl.CTA_TUPLE_IP {
 					parseIpTuple(reader, &s.Forward)
 				}
+				// TODO(thaJeztah): should this skipNfAttrValue ?
```

Comment:
i believe you can remove the TODO, setting wildcard is ok here since l (len) is not used here.

Reply:
removed